parasolr/query.py

```diff
@@ -158,6 +178,30 @@ def filter(self, *args, **kwargs) -> 'SolrQuerySet':
 
         return qs_copy
 
+    def facet(self, *args: str, **kwargs) -> 'SolrQuerySet':
+        """
+        Return a new SolrQuerySet with Solr faceting enabled and facet.field
+        parameter set. Does not yet support ranged faceting.
+
+        Subsequent calls will reset the facet.field to the last set of
+        args in the chain.
+
+        For example::
+
+            qs = queryset.facet('person_type', 'age')
+            qs = qs.facet('item_type')
+
+        would result in `item_type` being the only facet field.
+        """
+        qs_copy = self._clone()
+
+        # enable faceting
+        qs_copy.facet_opts['facet'] = True
+        # add args as a list to facet.field
+        qs_copy.facet_opts['facet.field'] = list(args)
```

Comment:
Isn't `args` already a list?

Reply:
`args` is actually a tuple, and given that we might as some point find ourselves modifying it, I thought it best to cast it to a mutable sequence explicitly. I'll note why explicitly, which I'd meant to do.

Reply:
I meant to check if it was a tuple. Makes sense, but a note would be helpful.